redi-connect-front/src/components/atoms/MediaIcons.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { SFC } from "react";
+import "./MediaIcons.scss";
+import linkedin from "../../assets/images/linkedin.svg";
+import twitter from "../../assets/images/twitter.svg";
+import instagram from "../../assets/images/instagram.svg";
+import medium from "../../assets/images/medium.svg";
+
+const SocialMediaIcons = () => {
+  const icons = [
+    {
+      name: linkedin,
+      link:
+        "https://www.linkedin.com/school/redi-school-of-digital-integration/",
+    },
+    { name: twitter, link: "https://twitter.com/redischool?lang=en" },
```

Comment:
"name" doesn't really contain the name. Let's change it to something more accurate, e.g. "image", "icon" or "iconContents"

Reply:
Change to image